Dictionaries/srp_OCRFixReplaceList.xml

```diff
@@ -61,15 +61,14 @@
     <!-- Two or more consecutive '"' to one '"' -->
     <RegEx find="&quot;{2,}" replaceWith="&quot;" />
     <!-- Fix zero and capital 'o' ripping mistakes -->
-    <RegEx find="([0-9])O" replaceWith="${1}0" />
-    <RegEx find="([0-9])\.O" replaceWith="$1.0" />
-    <RegEx find="\b0([A-Za-z])" replaceWith="O$1" />
+    <RegEx find="(?&lt;=[0-9]\.?)O" replaceWith="0" />
+    <RegEx find="\b0(?=[A-Za-z])" replaceWith="O" />
     <!-- Brisanje crte - na početku 1. reda (i kada ima dva reda) -->
     <RegEx find="\A- ?([A-ZČĐŠŽa-zčđšž0-9„'&quot;]|\.{3})" replaceWith="$1" />
     <RegEx find="\A(&lt;[i|b|u]&gt;)- ?" replaceWith="$1" />
     <RegEx find="  - " replaceWith=" -" />
     <!-- Brisanje razmaka iza crte - na početku 2. reda -->
-    <RegEx find="([\n](&lt;[i|b|u]&gt;)?)- ([0-9A-ZČĐŠŽčš„'&quot;&lt;])" replaceWith="$1-$3" />
+    <RegEx find="(\n(&lt;[i|b|u]&gt;)?)- ([A-ZČĐŠŽčš0-9„'&quot;&lt;])" replaceWith="$1-$3" />
```

Comment:
You don't need second group so `(\n(?:&lt;[i|b|u]&gt;)?)- ([A-ZČĐŠŽčš0-9„'&quot;&lt;])` would be :+1: 
and replace $3 with $2